### PR TITLE
Etd 160

### PR DESCRIPTION
**Pull request by michael-lts**
**Adding dash scripts, phase 1**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/ETD-160)

# What does this Pull Request do?
In this pr, existing scripts are added to the docker, refactored as necessary. Also includes telemetry tracing Test coverage above 80 currently, we can revisit 100% threshold if we can get the pragma skipping to work with flake8. Still missing from this, coming in next phase, is the job monitor (jobmon statements left in as comments fpr when util classes in place)

# How should this be tested?
Clone this repo and check out this branch
Follow readme for setting up telemetry prereq, starting up local docker.
Exec in to docker and run pytest
Once in qa, we can test a submission (has been done already on dev)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests? Y

# Interested parties
Tag (@ mention) interested parties

### CODE REVIEW

File: tests/unit/test_worker.py [hunk 1]
```diff
@@ -9,31 +13,122 @@ class MockResponse:
...
         # apply the monkeypatch for requests.get to mock_get
         monkeypatch.setattr(requests, "get", mock_get)
-        expected_msg = "REST api is NOT running."
-        worker = Worker()
+        expected_msg = "REST api is NOT runningx."
         msg = worker.call_api()
         assert msg != expected_msg
+
+    ''' # @unittest.skip("Need to get sftp key in place")
+    def test_send_to_dash(self, monkeypatch):
```
Comment (ives1227) [inline tests/unit/test_worker.py:46]: When this is working, do you think it would be better to consider it an integration test?

**Review by ives1227 — APPROVED** [review]

File: tests/unit/test_worker.py (hunk 1 above)
Comment (michael-lts) [reply tests/unit/test_worker.py:46]: @ives1227 , that was a misleading comment, I agree about making an itest instead, and that's reflected in changed comment

Comment (ives1227) [reply tests/unit/test_worker.py:46]: Great!

File: etd/worker.py [hunk 2]
```diff
@@ -8,18 +30,639 @@
...
+    dropboxUser = 'proquest'
 
     def __init__(self):
         self.version = os.getenv("APP_VERSION", "0.0.1")
 
     def get_version(self):
         return self.version
 
+    @tracer.start_as_current_span("send_to_dash_worker")
+    def send_to_dash(self, message):  # pragma: no cover
```
Comment (awoods) [inline etd/worker.py:72]: For all "# pragma: no cover" comments, can you also create one or more Jira tickets as a reminder to update the tests. The Jira ticket can then usefully be mentioned in a comment in the code.

File: etd/worker.py [hunk 3]
```diff
@@ -8,18 +30,639 @@
...
+
+        self.logger.debug(message)
+        filesDir = 'files'
+        csvEtds2Alma = os.path.join(filesDir, 'etds2alma.csv')
+        etds2AlmaOut = open(csvEtds2Alma, 'a+')
+
+        current_span = trace.get_current_span()
+        current_span.add_event("sending to dash")
+
+        print(filesDir)
```
Comment (awoods) [inline etd/worker.py:93]: Should this be a logger statement instead of `print`?

**Review by awoods — CHANGES_REQUESTED** [review]
Looks good. Two minor comments.

File: etd/worker.py (hunk 2 above)
Comment (michael-lts) [reply etd/worker.py:72]: Done:
https://jira.huit.harvard.edu/browse/ETD-205

File: etd/worker.py (hunk 3 above)
Comment (michael-lts) [reply etd/worker.py]: I removed this, it was extraneous

File: etd/worker.py (hunk 2 above)
Comment (michael-lts) [reply etd/worker.py:75]: Added comment to top of worker.py

**Review by awoods — APPROVED** [review]

**Pull request merged**